### PR TITLE
Reduces Midround Blob weight and ups threat cost & threshold.

### PR DESCRIPTION
**Pull request by Irradiation**
I know we've had that PR to standardize antags weight and whatnot, but ever since midround Blobs have been able to force the station into quarantine, things have gone slightly out of hand. I daresay that the quarantine happens fairly quickly, as midround Blobs get a bunch of free tiles to kickstart the progress to quarantine. 

Also I have my doubts regarding the theory behind "midround blobs being weak because they start exposed to space" and "the crew is armed and prepared", because the latter part isn't always true and negates the former. Then the Blob(s) quickly run out of control because nobody established emitter lines from space or 3+ guys with eguns came around to immediately zap the Blob(s) into submission. Then the shuttle gets canned because [QUARANTINE ENABLED, BUT THE AI WON'T TELL YOU THE NUKE CODE, ALSO THE BLOB ATE THE NUKE AND VERY POSSIBLY THE NUKE DISK HOLDER].

tl;dr I saw people die pls nerf

[gamemode][role][balance][tweak]

:cl:
 * tweak: Midround blobs are less likely to occur and will now cost more threat to launch.

### CODE REVIEW

**Comment by Kurfursten** [comment]
I'd be on board with this if you would make the core meteors penetrate deep into the station instead of sticking on the outside where they're exposed to space. 30 threat is just too much for something as weak as a spaceblob. Blobs take a 75% penalty to expansion in space which is why they're basically helpless against someone rolling up even with just the cap's laser. That's as much threat as a whole cult.

The roundstart blob which can plan a strategic location and pop on a highly desirable spot like telecoms inside a nest of protective r_walls is a serious threat, but not spaceblob.

**Comment by DeityLink** [comment]
How did I forget raising the midround blob costs? 🤔 I upped the roundstart one to 45.

**Comment by gbasood** [comment]
>The roundstart blob which can plan a strategic location and pop on a highly desirable spot like telecoms inside a nest of protective r_walls is a serious threat, but not spaceblob.

If we're using core meteors for this, why not just let players redirect their meteors?

**Comment by Kurfursten** [comment]
That would be a plus too if you wanna code it

**Comment by DeityLink** [comment]
hard mode: give the player a HUD indicator while in meteor blob form that lets them locate the direction toward the station.

**Comment by Irradiation** [comment]
Recent addendum: Blobs are still doing the "arrives now, puts the station on quarantine in 5 minutes, then slowly win the station easily because everybody is too busy reeling from other antags to do anything and we got a batch of new players that don't have a solid grasp of how to kill Blobs effectively" thing.
